src/main/scala/com/evolution/scache/LoadingCache.scala

```diff
@@ -169,46 +236,72 @@ private[scache] object LoadingCache {
                                       entryRef
                                         .get
                                         .flatMap {
-                                          case Left(_)      =>
-                                            0.tailRecM { counter =>
+                                          // Our deferred is still there
+                                          case _: EntryState.Loading[F, V] =>
+                                            0.tailRecM { counter1 =>
```

Comment:
we should remove this retry, so this is similar to successful path

Reply:
here we retry based on the whole `map` update, so it's vital to make sure bad key is removed